src/test/util-test.ts

```diff
@@ -35,4 +35,12 @@ describe('util', () => {
 			assert.equal(matches, true);
 		});
 	});
+	describe('isGlobalTSFile()', () => {
+		it('should match declaration files in node_modules', () => {
+			assert.equal(isGlobalTSFile('/node_modules/@types/not_node/not_node.d.ts'), true);
```

Comment:
This screws the semantics a bit, `isGlobalTSFile` is supposed to be a heuristic whitelist of known global declarations. Most declarations in `@types` are _not_ global. These should be fetched, but not by changing the definition of `isGlobalTSFile`